Added ipython3 support

MikeDacre opened the pull request:
I duplicated the IPython functions in autoload/screen.vim and plugin/screen.vim to include an 'IPython3' function that works identically to the :IPython function but starts ipython3 instead. I can confirm the :ScreenSend works as expected.

ervandew commented on autoload/screen.vim:
```diff
@@ -692,7 +692,18 @@ function! screen#IPython(...) " {{{
   exec 'ScreenShell' . bang . ' ipython'
 endfunction " }}}
+ 
+function! screen#IPython3(...) " {{{
```
Instead of adding a duplicate funtion that differs by 1 line, why not just update the existing function to take an addition argument before the `bang` arg to specify which ipython to run?

MikeDacre commented:
@ervandew Good thinking. I added a new variable that the user can set in their .vimrc 'g:ScreenIPython3'. If set to 1, ipython3 is used, otherwise the default behaviour is the same.
I added this to the documentation also.

ervandew commented:
Mind squashing those down to a single `Add ipython3 support` commit?

MikeDacre commented:
How's that?

ervandew commented:
Perfect, thanks!
